third_party/bigframes_vendored/google_cloud_bigquery/tests/unit/test_pyarrow_hlpers.py

```diff
@@ -0,0 +1,38 @@
+# Original: https://github.com/googleapis/python-bigquery/blob/main/tests/unit/test__pyarrow_helpers.py
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+pyarrow = pytest.importorskip("pyarrow", minversion="3.0.0")
+
+
+@pytest.fixture
+def module_under_test():
+    from third_party.bigframes_vendored.google_cloud_bigquery import _pyarrow_helpers
+
+    return _pyarrow_helpers
+
+
+def test_bq_to_arrow_scalars(module_under_test):
+    assert (
+        module_under_test.bq_to_arrow_scalars("BIGNUMERIC")
```

Comment:
shouldn't all types be unit tested?

Reply:
Maybe? That would mostly involve duplicating the source dictionary, though. Again, copied from upstream. We can make the patch there, potentially as part of go/pandas-gbq-and-bigframes-redundancy.

Since there are open questions,  I don't believe this PR needs to block GA. We can wait for that design to be resolved before merging this.